app/views/attachments/_nested_form.html.erb

```diff
@@ -45,7 +45,7 @@ an attachments_attributes= and not every model supports this we build a nested f
   <legend class="form--fieldset-legend" title="<%=l(:description_attachment_toggle)%>" onclick="toggleFieldset(this);">
     <a href="javascript:"><%=l(:label_attachment_plural)%></a>
   </legend>
-  <div <%= 'style="display: none;"' unless open %>>
+  <div <%= "style=\"display: none;\"".html_safe unless open %>>
```

Comment:
Incredible that this has been broken since the Rails 3 porting was done!

@floriank I'd probably use `raw` here.. it's a bit more succinct.